test/integration/cluster_logs_test.go

```diff
@@ -0,0 +1,41 @@
+// +build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"k8s.io/minikube/pkg/minikube/constants"
+	"k8s.io/minikube/test/integration/util"
+	"strings"
+	"testing"
+)
+
+func TestClusterLogs(t *testing.T) {
+	minikubeRunner := util.MinikubeRunner{BinaryPath: *binaryPath, T: t}
+	minikubeRunner.RunCommand("start", true)
+	minikubeRunner.CheckStatus("Running")
+
+	logsCmdOutput := minikubeRunner.RunCommand("logs", true)
+	//check for # of lines or check for strings
+	logFiles := []string{constants.RemoteLocalKubeErrPath, constants.RemoteLocalKubeOutPath}
+	for _, logFile := range logFiles {
+		if !strings.Contains(logsCmdOutput, logFile) {
+			t.Fatalf("Error in logsCmdOutput, expected to find %s : %s. Output: %s", logFile, logsCmdOutput)
```

Comment:
Fixed, thanks.

On Tue, May 31, 2016 at 5:39 PM, dlorenc <notifications@github.com> wrote:

> In test/integration/cluster_logs_test.go
> <https://github.com/kubernetes/minikube/pull/146#discussion_r65286664>:
>
> > +	"k8s.io/minikube/test/integration/util"
> > +	"strings"
> > +	"testing"
> > +)
> > +
> > +func TestClusterLogs(t *testing.T) {
> > +	minikubeRunner := util.MinikubeRunner{BinaryPath: *binaryPath, T: t}
> > +	minikubeRunner.RunCommand("start", true)
> > +	minikubeRunner.CheckStatus("Running")
> > +
> > +	logsCmdOutput := minikubeRunner.RunCommand("logs", true)
> > +	//check for # of lines or check for strings
> > +	logFiles := []string{constants.RemoteLocalKubeErrPath, constants.RemoteLocalKubeOutPath}
> > +	for _, logFile := range logFiles {
> > +		if !strings.Contains(logsCmdOutput, logFile) {
> > +			t.Fatalf("Error in logsCmdOutput, expected to find %s : %s. Output: %s", logFile, logsCmdOutput)
>
> Looks like there's an extra %s.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/minikube/pull/146/files/97d7fcc3a5a4c4161d74b5935e806d0dd205112e#r65286664>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB3HqjljUjTG1NdRid9ir2k2ebXOznCuks5qHNS-gaJpZM4IrFYW>
> .
>